app/views/layouts/forever_style_guide/_navigation.erb

```diff
@@ -8,7 +8,7 @@
         <span class="icon-bar"></span>
       </button>
       <%= link_to forever_style_guide_path, class: "navbar-brand" do %>
-        <%= image_tag("forever_style_guide/forever-logo.png", alt: "Forever Logo", class:"navbar-brand-logo") %>
+        <%= image_tag image_url("forever_style_guide/forever-logo.jpg", alt: "Forever Logo", class:"navbar-brand-logo") %>
```

Comment:
What is the image_url doing here?  Seems like it is probably swallowing the alt and class keys, since those are intended for the image_tag.